routing/routing_tests/index_graph_tools.cpp

```diff
@@ -151,47 +151,47 @@ bool TestIndexGraphTopology::FindPath(Vertex start, Vertex finish, double & path
   // for the end of our path and another loop at start for the bidirectional search.
   auto const startFeatureId = static_cast<uint32_t>(edgeRequests.size());
   AddDirectedEdge(edgeRequests, start, start, 0.0);
+  // |startSegment| corresponds edge from |start| to |start| which has featureId |startFeatureId|
+  // 1 segment with segmentIdx 0 and it's loop so direction does not matter.
```

Comment:
Could please rephrase this please:
... which has featureId |startFeatureId| 1 segment with ...

Reply:
sure, done